src/xxhash64.h

```diff
@@ -0,0 +1,203 @@
+// Copyright © 2016 Stephan Brumme. All rights reserved, see http://create.stephan-brumme.com/disclaimer.html
```

Comment:
Also add link to: https://github.com/Cyan4973/xxHash/blob/dev/LICENSE
Their BSD clause requires this, for source redistribution.

Reply:
This code is not from Cyan4973/xxHash